cobalt/configuration/configuration.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COBALT_CONFIGURATION_CONFIGURATION_H_
+#define COBALT_CONFIGURATION_CONFIGURATION_H_
+
+#include <string>
+
+#include "base/logging.h"
+#include "base/notreached.h"
+#include "starboard/extension/configuration.h"
+
+namespace base {
+template <typename T>
+struct DefaultSingletonTraits;
+}
+
+namespace cobalt {
+namespace configuration {
+
+constexpr char kUserOnExitStrategyNoExit[] = "noexit";
+constexpr char kUserOnExitStrategyStop[] = "stop";
+constexpr char kUserOnExitStrategySuspend[] = "suspend";
+
+// The Configuration changes certain Cobalt features as specified by the
+// platform. This class picks up values set in the
+// CobaltConfigurationExtensionApi if it is implemented by the platform and
+// will otherwise use default configurations.
+class Configuration {
```

Comment:
"Configuration changes certain Cobalt features" -- is this so? FWICT `Configuration` seems to be a wrapper around the similarly named Starboard extension. If so I'd suggest calling this class  "SbConfigurationWrapper" or "SbConfigurationExtensionWrapper".